x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_management/linked_dashboards_view.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { DashboardRenderer } from '@kbn/dashboard-plugin/public';
+import { Streams } from '@kbn/streams-schema';
+import {
+  EuiListGroup,
+  EuiListGroupItem,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+import type { SanitizedDashboardAsset } from '@kbn/streams-plugin/server/routes/dashboards/route';
+import { i18n } from '@kbn/i18n';
+import { useDashboardsFetch } from '../../../hooks/use_dashboards_fetch';
+
+export function LinkedDashboardsView({ definition }: { definition: Streams.all.GetResponse }) {
```

Comment:
I can't find the way to add a dashboard to a group stream and the group member's dashboards are not surfaced there. How can I view group stream dashboards ?

Reply:
Adding dashboard (and other assets) is meant to be a Streams wide thing, so you use the same API as for adding a dashboard to an ingest stream. 

It's true that we don't surface Dashboards from member streams, not sure if we want to do that later on.
cc @roshan-elastic

Reply:
We'll need to come back together with overall Streams design about how users should link up Dashboards again to any Stream.